source/gen/FlashOWare.Generators/Generators/Enumerations/EnumInterceptorGenerator.cs

```diff
@@ -102,26 +102,26 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
 			document.WriteLine('{');
 			document.Indent++;
 
-			IEnumerable<IGrouping<string, EnumMethodInvocation>> group = source.GroupBy(static value => value.Name);
+			IEnumerable<IGrouping<string, EnumMethodInvocation>> group = source.GroupBy(static value => $"{value.Name}:{value.Enum.Name}");
 
 			foreach (var invocations in group)
```

Comment:
I made a mistake here ... since `group` is of type `IEnumerable<T>`, it should have a plural name instead: `groups`.
Would you mind fixing my mistake?

Also would you mind using explicit types over `var` for the following 3 `var`s? Also a previous mistake of mine.
In the `.editorconfig` I defnied `csharp_style_var_when_type_is_apparent = true:suggestion`.

(I'm reviewing my own code now and I am quite displeased 😉)